src/components/restaurant-carousel/RestaurantCarousel.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { Carousel } from "@mantine/carousel";
+import "./index.scss";
+
+const RestaurantCarousel = () => {
+  return (
+    <div className="restaurant-carousel">
+      <div className="restaurant-carousel__box">
+        {/* <div className="restaurant-carousel__logo"> */}
```

Comment:
Please, delete this if this is not neccesary.

Reply:
Resolved